localization/ndt_scan_matcher/README.md

```diff
@@ -255,3 +255,79 @@ initial_pose_offset_model is rotated around (x,y) = (0,0) in the direction of th
+| `nearest_voxel_transformation_likelihood` | the score of how well the map aligns with the sensor points                            | the score is **smaller** than `score_estimation.converged_param_nearest_voxel_transformation_likelihood` (only in the case of `score_estimation.converged_param_type` is 1=NEAREST_VOXEL_TRANSFORMATION_LIKELIHOOD)                                                                                                                                                      |                                          | no                                                                       | yes |
+| `distance_initial_to_result`              | the distance between the position before convergence processing and the position after | the distance is **longer** than 3                                                                                                                                                                                                                                                                                                                                        | none                                     | no                                                                       |
+| `execution_time`                          | the time for convergence processing                                                    | the time is **longer** than `validation.critical_upper_bound_exe_time_ms`                                                                                                                                                                                                                                                                                                | none                                     | no                                                                       |
+| `skipping_publish_num`                    | the number of times rejected estimation results consecutively                          | the number of times is **larger** than 0 and **smaller** than 5                                                                                                                                                                                                                                                                                                          | the number of times is **larger** than 5 | -                                                                        |
```

Comment:
For some reason pre-commit is destroying the table :cry: 
https://github.com/autowarefoundation/autoware.universe/pull/5076/commits/ce8adef9fa19d08d7b41b5f0aea967f43e028186

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/5076/commits/3ccfa245e549be62cb3f89d98e6d7c13bfec1c4c